buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -439,14 +440,18 @@ def workflow(SDK_VERSION, SPEC, SHAS, OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO,
                 PARALLEL = "Dynamic"
                 NUM_MACHINES = "4"
             }
+
+            // generate child test jobs
+            def GENERATE_JOBS = params.AUTOMATIC_GENERATION ?: false
```

Comment:
`AUTOMATIC_GENERATION` is a string, `false` is a bool. Test Auto gen job is expecting a bool.

Reply:
Or...does this mean if params.AUTOMATIC_GENERATION is set to a value?

Reply:
I did not realize that params.AUTOMATIC_GENERATION is a string (not a boolean) when I created this PR. However, this will not cause any issue because Test Auto gen job does check the value and do the conversion. For example https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/testJobTemplate#L405. 
Also, in my test, the pipeline worked as expected (2nd case) https://github.com/eclipse-openj9/openj9/pull/14345#issuecomment-1019288342

Even though the behavior is correct, it may a good idea to cover it here as well to avoid future confusion. I will create a PR.